compiler/x/env/OMRCPU.cpp

```diff
@@ -27,9 +27,42 @@
 #include "infra/Flags.hpp"
 #include "x/runtime/X86Runtime.hpp"
 
+namespace OMR
+{
+
+namespace X86
+{
+
+CPU::CPU() :
+   OMR::CPU()
+   {
+   // Only enable the features that compiler currently uses
+   uint32_t enabledFeatures [] = {OMR_FEATURE_X86_FPU, OMR_FEATURE_X86_CX8, OMR_FEATURE_X86_CMOV,
+                                  OMR_FEATURE_X86_MMX, OMR_FEATURE_X86_SSE, OMR_FEATURE_X86_SSE2,
+                                  OMR_FEATURE_X86_SSSE3, OMR_FEATURE_X86_SSE4_1, OMR_FEATURE_X86_POPCNT,
+                                  OMR_FEATURE_X86_AESNI, OMR_FEATURE_X86_OSXSAVE, OMR_FEATURE_X86_AVX,
+                                  OMR_FEATURE_X86_HLE, OMR_FEATURE_X86_RTM};
+
+   for (int i = 0; i < OMRPORT_SYSINFO_FEATURES_SIZE; i++)
+      {
+      _featureMasks[i] = 0;
+      }
+
+   for (uint32_t feature : enabledFeatures)
```

Comment:
I didn't think range-based for loops were supported by all build compilers (see https://github.com/eclipse/omr/blob/master/doc/SupportedC%2B%2BFeatures.md#unsupported-c11-features)